src/pages/iou/request/step/IOURequestStepConfirmation.tsx

```diff
@@ -354,9 +358,13 @@ function IOURequestStepConfirmation({
             }
 
             formHasBeenSubmitted.current = true;
-
             playSound(SOUNDS.DONE);
 
+            if (isDistanceRequest && !isMovingTransactionFromTrackExpense) {
```

Comment:
Moving this `createDistanceRequest` call here caused an error in https://github.com/Expensify/App/issues/47042 when tracking a distance expense as we end up calling `CreateDistanceRequest` here instead of `trackExpense`.